Frameworks/QuartzCore/CALayer.mm

```diff
@@ -281,12 +290,13 @@ static void DoDisplayList(CALayer* layer) {
     _name = nil;
 
     // Create our backing layer proxy
-    _layerProxy = GetCACompositor()->CreateLayerProxy(xamlElement ? [xamlElement comObj] : nullptr);
+    auto rawElement = reinterpret_cast<ABI::Windows::UI::Xaml::IFrameworkElement*>(winrt::get_abi(xamlElement));
```

Comment:
Because I wrote this before I invented to_insp. Will fix...

---
In reply to: [109241662](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109241662) [](ancestors = 109241662)